llvm/lib/Transforms/Scalar/ConstraintElimination.cpp

```diff
@@ -1392,6 +1396,31 @@ static bool checkAndReplaceCondition(
   return false;
 }
 
+static bool checkAndReplaceMinMax(MinMaxIntrinsic *MinMax, ConstraintInfo &Info,
+                                  unsigned NumIn, unsigned NumOut,
+                                  Instruction *ContextInst, DominatorTree &DT,
```

Comment:
DT unused.

Reply:
Rebase over https://github.com/llvm/llvm-project/commit/ebb853fbe5f67ab42d78de45bbcb7b32b6323fb1 please, and remove other unused params.

Reply:
Done.